pkg/cmd/roachtest/tests/drt.go

```diff
@@ -211,6 +200,29 @@ func (ep *tpccChaosEventProcessor) checkMetrics(
 	return nil
 }
 
+func (ep *tpccChaosEventProcessor) queryPrometheus(
+	ctx context.Context, l *logger.Logger, q string, ts time.Time,
+) (val model.Value, err error) {
+	rOpts := base.DefaultRetryOptions()
+	rOpts.MaxRetries = 5
+	var warnings promv1.Warnings
```

Comment:
I don't know what this type is but I'm guessing it an int of some sort so default is 0 and shouldn't cause issues with the check on line 220 if it never gets assigned a value right? Thinking of the case all 5 retries happen.

Reply:
nah if this happens, err will be nil, so we spit out this error instead and never retry